### PR TITLE
Fixed bar plot envelope dashboard spacing

### PR DESCRIPTION
**Pull request by SamRagusa**
- The bars that try to plot an envelope of some data had gaps in them (I thought I had fixed this before, but I applied the fix in the wrong place originally)

### CODE REVIEW

File: endaq/plot/dashboards.py [hunk 1]
```diff
@@ -128,19 +128,6 @@ def rolling_enveloped_dashboard(
...
-
-        rolling_n = channel_data.rolling(window)
-        min_max_tuple = (rolling_n.min()[::stride], rolling_n.max()[::stride])
-        min_max_equal = min_max_tuple[0] == min_max_tuple[1]
-
-        is_nan_min_max_mask = np.logical_not(
-            np.logical_and(
-                pd.isnull(min_max_tuple[0]),
-                pd.isnull(min_max_tuple[1])))
-
```
Comment (CrepeGoat) [inline endaq/plot/dashboards.py:143]: moving this chunk has no functional effect, right? It's just for organization?

**Review by CrepeGoat — APPROVED** [review]
looks fine

File: endaq/plot/dashboards.py (hunk 1 above)
Comment (CrepeGoat) [reply endaq/plot/dashboards.py:143]: or I guess the use of `channel_data` might rely on the new index that was set?

**Review by StokesMIDE — APPROVED** [review]

File: endaq/plot/dashboards.py (hunk 1 above)
Comment (SamRagusa) [reply endaq/plot/dashboards.py:143]: The reason it matters is that the `rolling` method of the channel DataFrame was being used _before_ the index was set properly.   The other issue that was fixed is that there was a line calling `set_index` without setting `inplace=True`.

**Pull request merged**